deny.toml

```diff
@@ -150,6 +150,14 @@ skip = [
 
     # some non-servo crates still use 0.14
     "glow",
+
+    # duplicated by webdriver
+    "h2",
+    "headers",
+    "headers-core",
+    "http",
+    "http-body",
+    "hyper",
```

Comment:
It seems likely that we are going to need to write our own webdriver server to avoid problems like this. We already have issues such as #33486 due to the use of `warp` in `webdriver`.